### PR TITLE
single pass through each storage to populate info

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Problem
speeding up index generation on startup

#### Summary of Changes
go from 2 passes 1 through each account in each append vec

Fixes #

### CODE REVIEW

File: accounts-db/src/accounts_db.rs [hunk 1]
```diff
@@ -15847,6 +15836,7 @@ pub mod tests {
                 (&0, 2, 1280)
             );
         }
+        accounts.accounts_index.set_startup(Startup::Normal);
```
Comment (HaoranYi) [inline accounts-db/src/accounts_db.rs:15839]: Looks like `cargo fmt` didn't like this line. 
Is this line relevant to this pr change?

Comment (jeffwashington) [reply accounts-db/src/accounts_db.rs:15839]: Yes, required. I fixed it already.

File: accounts-db/src/accounts_db.rs [hunk 2]
```diff
@@ -9481,36 +9488,6 @@ impl AccountsDb {
...
-            Measure::start("storage_size_accounts_map_flatten_time");
-        if !accounts_map.is_empty() {
-            let mut info = storage_info.entry(store_id).or_default();
-            info.stored_size += storage_info_local.stored_size;
-            info.count += storage_info_local.count;
-        }
-        storage_size_accounts_map_flatten_time.stop();
-
-        let mut timings = timings.lock().unwrap();
-        timings.storage_size_accounts_map_us += storage_size_accounts_map_time.as_us();
```
Comment (HaoranYi) [inline accounts-db/src/accounts_db.rs:9510]: nit: looks like these two timings can be removed, don't need to be done in this pr though. 
it can be done with a follow up pr.

**Review by HaoranYi — APPROVED** [review]
lgtm
